java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_1/Orchestra.java

```diff
@@ -0,0 +1,23 @@
+package com.zoolatech.lecture3.tasks._1;
+
+import java.util.ArrayList;
+
+public class Orchestra {
+    ArrayList<Musicians> listOfMusicians = new ArrayList<>();
+
+    public void add(Musicians musicians) {
+        listOfMusicians.add(musicians);
+        System.out.println(musicians + " joined first group");
+    }
+
+    public void remove(Musicians musicians) {
+        listOfMusicians.remove(musicians);
+        System.out.println(musicians + " left first group");
+    }
+
+    public void playAll() {
+        for (Musicians m : listOfMusicians) {
+            m.instrument.play();
```

Comment:
It would be better to add some method to a musician that will play music instead of accessing a field. It would also allow you to make this field private.

Reply:
fixed